app/Console/Kernel.php

```diff
@@ -24,7 +25,9 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        // $schedule->command('inspire')->hourly();
+        $schedule->call(new TrackMismatchRatio())
+            ->description("Tracking task that runs every minute")
+            ->everyMinute();
```

Comment:
why is it every minute? isn't that too frequent? do we need that level of granularity? i may have missed or forgotten anything related to the frecuency of the scheduler.

Reply:
Good question, let me find out. Most probably it isn't even needed every minute.